diff --git a/include/c11/threads_posix.h b/include/c11/threads_posix.h
index 802526a..14f269d 100644
--- a/include/c11/threads_posix.h
+++ b/include/c11/threads_posix.h
@@ -382,7 +382,7 @@ tss_set(tss_t key, void *val)
 
 /*-------------------- 7.25.7 Time functions --------------------*/
 // 7.25.6.1
-#ifndef HAVE_TIMESPEC_GET
+/*#ifndef HAVE_TIMESPEC_GET
 static inline int
 timespec_get(struct timespec *ts, int base)
 {
@@ -394,3 +394,4 @@ timespec_get(struct timespec *ts, int base)
     return 0;
 }
 #endif
+*/
